dial-cookbook/examples/how_to_call_image_to_text_applications.ipynb

```diff
@@ -6,11 +6,11 @@
    "source": [
     "# How to call image-to-text DIAL applications\n",
     "\n",
-    "This notebook covers how to call image-to-text applications via [DIAL API chat/completions](https://epam-rail.com/dial_api#/paths/~1openai~1deployments~1%7BDeployment%20Name%7D~1chat~1completions/post) call.\n",
+    "From [this notebook](https://github.com/epam/ai-dial/blob/main/dial-cookbook/examples/how_to_call_image_to_text_applications.ipynb), you will learn how to call image-to-text DIAL applications via [DIAL API chat/completions](https://epam-rail.com/dial_api#/paths/~1openai~1deployments~1%7BDeployment%20Name%7D~1chat~1completions/post) call.\n",
     "\n",
-    "**DIAL application** is a general term, which encompasses model adapters and application with custom logic.\n",
+    "**DIAL Application** is a general term, which encompasses model Adapters and Application with any custom logic.\n",
```

Comment:
It's strange to have **A**pplication and **A**dapters capitalized. They are not proper names.

Reply:
fixed